README.md

```diff
@@ -251,13 +251,13 @@ method takes two arguments:
 A value that implements the Traversable specification must also
 implement the Functor and Foldable specifications.
 
-1. `t(u.sequence(f.of))` is equivalent to `u.map(t).sequence(g.of)`
+1. `t(u.traverse(x => x, F.of))` is equivalent to `u.traverse(t, G.of)`
 for any `t` such that `t(a).map(f)` is equivalent to `t(a.map(f))` (naturality)
```

Comment:
Could someone explain this to me? Unless I missed something `F` and `G` have not been introduced by this point. I see that they are mentioned below.

I struggle to see how an expression involving `F` (but not `G`) can be equivalent to an expression involving `G` (but not `F`). How is this possible?